### PR TITLE
Add introduction, ignore/remove generated file

### PR DESCRIPTION
**Pull request by ericzolf**
* move some of the introduction from the roles' readme to the main one
* complete it with a few generic words about purpose and content of the
  repo
* ignore all generated files (html or pdf) and remove existing one.

### CODE REVIEW

File: roles/README.adoc
Comment (acozine) [inline roles/README.adoc:4]: To match the main page above, which abbreviates Good Practices for Ansible as GPA:
```suggestion
GPA Roles repositories.
```

**Review by acozine — COMMENTED** [review]
A couple of small wordsmithing suggestions.

File: roles/README.adoc
Comment (ericzolf) [reply roles/README.adoc]: I removed it altogether, the introduction anyway didn't fit the context.

**Comment by ericzolf** [comment]
I've addressed the comments from @acozine but she's not member of the team so I need someone from @redhat-cop/automation-cop-members to approve.
Alicia, if you want to get write access, you need first to become member of redhat-cop (I think you need to request it and it'll be granted, if not let me know).

**Review by sean-m-sullivan — APPROVED** [review]
I think these look like good changes. Propose at least one more review before merge though.

**Comment by ericzolf** [comment]
We'll wait until Wednesday and validate the merge in the CoP meeting.

**Comment by jscar-hawk** [comment]
I'll update and take note of what the Wednesday Meeting decides.

**Pull request merged**
